charts/chronograf/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: chronograf
-version: 1.1.19
+version: 1.1.20
```

Comment:
just need to bump this forward.  We've had another merge since...

Reply:
bumped in 9310ec2